src/Time/StepChoosers/ErrorControl.hpp

```diff
@@ -26,6 +26,7 @@ class GlobalCache;
 
 namespace StepChoosers {
 namespace Tags {
+template <typename EvolvedVariableTag>
```

Comment:
should this feature be mentioned in the documentation for `ErrorControl` or somewhere else and maybe what the intended use-case is?